api/classes/base_model.js

```diff
@@ -1,13 +1,19 @@
-var Bookshelf = require('./database').Bookshelf;
+"use strict";
 
-var instanceProps = {
-  column: function(columnName, alias, escapeString) {
-    var prefix = escapeString ? '"' : '';
-    var column = prefix + this.tableName + prefix + '.' + prefix + columnName + prefix;
-    return alias ? column + ' AS ' + prefix + alias + prefix : column;
+const Bookshelf = require(`./database`).Bookshelf;
+
+const instanceProps = {
+  column(columnName, alias, escapeString) {
+    const prefix = escapeString ? `"` : ``;
+    const column = `${prefix}${this.tableName}${prefix}` +
+                   `.` +
```

Comment:
I use both because of character length per line constraints.
Do you know how to put them on separate lines with backticks so that it won't consider the newlines @Pomax?